### PR TITLE
[code] build stable code image `1.72.3`

### PR DESCRIPTION
**Pull request by mustard-mh**
## Description

- Upstream changes
- https://github.com/gitpod-io/openvscode-server/commit/b46c7b1d62b8a86ccd08948ec201d38c7734bb59
- https://github.com/gitpod-io/openvscode-server/commit/a1873c7c6dd518a46d25738d568716a16d364b27
- https://github.com/gitpod-io/openvscode-server/commit/5e03841f0f85c4d00bfeb771f9b5895626a82c15

## Related Issue(s)

- Related https://github.com/gitpod-io/gitpod/issues/13410

## How to test

- Switch to VS Code Browser Insiders in settings.
- Start a workspace.
- Test the following:
  - [x] terminals are preserved and resized properly between window reloads
  - [x] WebViews are working
  - [x] extension host process: check language smartness and debugging 
  - [x] extension management (installing/uninstalling)
  - [x] install the [VIM extension](https://open-vsx.org/extension/vscodevim/vim) to test web extensions
  - that user data is synced across workspaces as well as on workspace restarts, especially for extensions
     - [x] extensions from `.gitpod.yml` are not installed as sync
     - [x] extensions installed as sync are actually synced to all new workspaces
  - [x] settings should not contain any mentions of MS telemetry
  - [x] WebSockets and workers are properly proxied
     - [x] diff editor should be operatable
     - [x] trigger reconnection with `window.WebSocket.disconnectWorkspace()`, check that old WebSockets are closed and new opened of the same amount
  - [x] workspace specific commands should work, i.e. F1 → type `Gitpod` prefix
  - [x] that a PR view is preloaded when opening a PR URL
  - [x] test `gp open` and `gp preview`
  - [x] test open in VS Code Desktop, check `gp open` and `gp preview` in task/user terminals
  - [x] telemetry data is collected in [Segment](https://app.segment.com/gitpod/sources/staging_untrusted/debugger)

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [x] /werft analytics=segment|TEZnsG4QbLSxLfHfNieLYGF4cDwyFWoe
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

**Comment by mustard-mh** [comment]
/hold

To test first

**Comment by mustard-mh** [comment]
@filiptronicek Could you approve this PR? I'll test it after werft build succeeds, and then unhold it 🙏

**Review by filiptronicek — APPROVED** [review]

**Comment by filiptronicek** [comment]
@mustard-mh you got it! Thanks for taking over this patch release, I was waiting for Microsoft to release (thought they might add more changes to the release branch), but it doesn't seem like they will and it seems more that they forgot to create a release 😆.

**Comment by mustard-mh** [comment]
/unhold

Works 🎉 
image

**Pull request merged**
